docs/deploy/pg_analytics.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: pg_analytics
```

Comment:
Do we still want to keep this around despite deprecating it? Perhaps it should at least go in the analytics/deprecated?